pkg/client/clientset/versioned/fake/register.go

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 package fake
 
 import (
-	openebsv1 "github.com/openebs/jiva-operator/pkg/apis/openebs/v1"
+	jivaapi "github.com/openebs/jiva-operator/pkg/apis/openebs/v1"
```

Comment:
We can ignore the files in pkg/client/clientset as these are auto generated files.

Reply:
sure